aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1584,6 +1585,15 @@ void scaled_gemm(
     computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_BIAS_DATA_TYPE, ScalarTypeToCudaDataType(bias_dtype));
   }
 
+  if (mat1_scale_dtype == kFloat8_e8m0fnu && mat2_scale_dtype == kFloat8_e8m0fnu) {
+#if CUDA_VERSION >= 12080
```

Comment:
Does it make sense to gate a ROCM version similarly here CC @jeffdaily 
Otherwise is it just an #if 1 due to different version interpretation on ROCM?

Reply:
For better or worse we hipify CUDA_VERSION to TORCH_HIP_VERSION which effectively is our ROCm distro version.  Frankly the comparison is bogus.  TODO for later.  Here it would be 603 >= 12080 which is false so ROCm will fall back to the TORCH_CHECK(false) here.

Reply:
So we should have some "MIN_MX_SUPPORTED_ARCH" (name pending) defined above that we compare against?

Reply:
```suggestion
#if !defined(USE_ROCM) && CUDA_VERSION >= 12080
```
We could have a seperate version check for ROCm

Reply:
It would make ROCm devs' lives easier if pytorch didn't do any CUDA version checks either in C++ or python and instead had feature macros or attrs.  What the cuda version checks are doing is guarding against new features/APIs.  Anywhere in pytorch that checks a cuda version we have to scrutinize and put either a defined(USE_ROCM) [enable feature for rocm] or !defined(USE_ROCM) [disable].